src/SME.SGP.Aplicacao/Commands/PendenciaDiasNaoLetivos/VerificarPendenciaAulaDiasNaoLetivos/VerificarPendenciaAulaDiasNaoLetivosCommandHandler.cs

```diff
@@ -96,10 +96,10 @@ private async Task<string> ObterDescricao(AulaReduzidaDto aula, TipoPendencia ti
             var componenteCurricular = await ObterComponenteCurricular(long.Parse(aula.DisciplinaId));
             var mensagem = new StringBuilder();
 
-            mensagem.AppendLine(tipoPendencia.Name());
+            mensagem.AppendLine($"<i>{tipoPendencia.Name()}</i>");
             mensagem.AppendLine("<br />");
-            mensagem.AppendLine($"<i>Componente Curricular:</i> <b>{componenteCurricular?.Nome ?? aula.DisciplinaId}</b><br />");
-            mensagem.AppendLine($"<i>Professor</i>: <b>{aula.Professor}({aula.ProfessorRf})</b><br />");
+            mensagem.AppendLine($"<i>Componente Curricular:</i> {componenteCurricular?.Nome ?? aula.DisciplinaId}<br />");
+            mensagem.AppendLine($"<i>Professor</i>: {aula.Professor}({aula.ProfessorRf})<br />");
```

Comment:
No exemplo postado pelo Marlon esta assim
```suggestion
            mensagem.AppendLine($"<i>Componente Curricular: {componenteCurricular?.Nome ?? aula.DisciplinaId}</i><br />");
            mensagem.AppendLine($"<i>Professor: {aula.Professor}({aula.ProfessorRf})</i><br />");
```